components/Discussion/Discussion.js

```diff
@@ -31,16 +32,23 @@ const Discussion = ({
       : discussionId === GENERAL_FEEDBACK_DISCUSSION_ID
       ? 'DATE'
       : 'AUTO')
+  const tag = query.tag
 
   const depth = board ? 1 : DEFAULT_DEPTH
 
   return (
     <div data-discussion-id={discussionId}>
       {!rootCommentOverlay && (
         <>
+          <TagFilter
+            discussionId={discussionId}
+            depth={depth}
+            orderBy={orderBy}
+          />
```

Comment:
Since `withDiscussionComments` it needs to get all the query relevant props like `DiscussionCommentComposer` and `Comments` below: currently missing `focusId`, `parentId` and `tag`. Otherwise two queries are fired:
<img width="898" alt="Screenshot 2021-11-01 at 14 58 55" src="https://user-images.githubusercontent.com/410211/139684429-8fe93b98-97b6-42bc-a977-75a8185e7b68.png">
<img width="896" alt="Screenshot 2021-11-01 at 14 58 25" src="https://user-images.githubusercontent.com/410211/139684435-c35eb307-4eac-4f37-be07-6b29b283ad51.png">